### DISCUSSION TITLE
Tree size resized 50% width when closing buffer with `:q` or `:bd`

### DISCUSSION THREAD

**Discussion by ckt114** [nvim-tree/nvim-tree.lua#1260] [Q&A] [answered]
### Description

When opening a file and close that file with `:q` or `:bd` and then open another file the tree window size splits into 50% width of the screen. It doesn't keep its original size.

### Neovim version

```text
NVIM v0.7.0
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

macOS 12.3.1

### nvim-tree version

82ec79a

### Minimal config

```lua
Used the nvt-min.lua file and same behavior.
```

### Steps to reproduce

1. nvim -nu /tmp/nvt-min.lua
2. :NVimTreeOpen
3. Open a file
4. Close that file buffer with `:q` or `:bd`
5. Open another file
6. Tree window is resized to 50% of screen

### Expected behavior

Tree window to stay the same.

### Actual behavior

Tree window resized to 50% of screen.

**Comment by ckt114** [comment]
This happens as well when the last buffer is closed and then opening another file.

**Comment by alex-courtis** [comment]
Replicated with minimal config, but not with my config.

Occurs when `actions.open_file.resize_window = false` (default)

**Comment by alex-courtis** [comment, marked as answer]
This is working as intended. Please set `actions.open_file.resize_window = true` if you desire different behaviour.

See `:help actions.open_file.resize_window`
